policy/modules/system/userdomain.te

```diff
@@ -206,6 +206,10 @@ optional_policy(`
 	telepathy_filetrans_home_content(userdomain)
 ')
 
+optional_policy(`
+	tuned_dbus_chat(userdomain)
```

Comment:
I wonder if this should really be allowed to the whole `userdomain`... Communicating with tuned sounds like something only sysadm_t (maybe also staff_t?) should be allowed to do.

Reply:
It is only sysadm_t, staff_u only in the sysadm_r role. So the tests needs updating, too, apparently not everything which is tested is reasonable.